### PR TITLE
updating_readme

### PR DESCRIPTION
**Pull request by mkazmer**
My first PR to update those couple things in README!!!!!!!!  OMG!!!!!!!!!!!!!!!!!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,21 @@
     * If branch does exist remotely
         * `git push`
 
-**TODO** - Add documentation for updating branch with latest changes to master
+* Updating branch w/latest changes to master
+    * Update your local master branch
+        * `git checkout master` - Switches from current branch to master
+        * `git fetch -p origin` - Brings branches and their commits from remote.
```
Comment (acousineau) [inline README.md]: Updates local references of branches with their remote counterparts

File: README.md [hunk 2]
```diff
@@ -11,9 +11,21 @@
...
+    * Update feature branch with local master
+        * `git checkout <feature-branch>` - Switch back to branch
+        * `$ git merge master` - Merge changes from updated local master
 
-**TODO** - Add documentation for ensuring master is up to date locally
+
+* Ensuring master is up to date locally
+    * Make sure you have master currently checked out
+        * **If NOT** `git checkout master`
+    * `git push` - Does both `git fetch` & `git merge` in one command
```
Comment (acousineau) [inline README.md]: `git pull`

**Comment by acousineau** [comment]
Testing a comment.....

File: README.md (hunk 2 above)
Comment (mkazmer) [reply README.md]: ....I'm a dummy...    :|

**Pull request merged**